packages/registry/src/utils/verify-string.ts

```diff
@@ -0,0 +1,5 @@
+export function isAlphanumeric(stringToVerify: string): boolean {
```

Comment:
Alphanumeric means consisting of letters and numbers, but I have no suggestions for a better name. Maybe we should have `isValidSimulationTitle` and `isValidComponentName`, and the latter will have a different validation rule that only allows letters and numbers, and no spaces or punctuation. What do you think?